server/http.go

```diff
@@ -411,42 +692,67 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	client, _, err := p.getActiveClient(user)
-	if err != nil {
-		p.API.LogWarn("Error getting the client", "err", err)
-		return
+	topic := req.Topic
+	if topic == "" {
+		topic = DefaultMeetingTopic
 	}
 
-	meeting, err := client.CreateMeeting(zoomUser, defaultMeetingTopic)
-	if err != nil {
-		p.API.LogWarn("Error creating the meeting", "err", err)
+	var meetingID int
+	var createMeetingErr error = nil
+	userPMISettingPref, getUserPMISettingErr := p.getPMISettingData(user.Id)
+	if getUserPMISettingErr != nil {
```

Comment:
What's the reason for creating a bunch of different variable names for different error occurrences?

Reply:
Now using `err`